satellite/console/consoleweb/server.go

```diff
@@ -387,7 +681,18 @@ func (server *Server) serveError(w http.ResponseWriter, r *http.Request, status
 	}
 }
 
-// grapqlHandler is graphql endpoint http handler function
+func (server *Server) serveJSONError(w http.ResponseWriter, status int, err error) {
+	w.WriteHeader(status)
+
+	server.log.Error(Error.Wrap(err).Error())
```

Comment:
Not all the status codes must be logged as an error.

Codes >= 500 must be logged as an 'ERROR` message', the other ones  should be logged as a `DEBUG` message